ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -118,13 +118,22 @@ Item {
     Repeater {
         id: chatRepeater
         model: parentModule && parentModule.model
+
         delegate: Loader {
             id: chatLoader
-            // Channels are not loaded by default and only load when first put active
-            active: model.active
-            visible: active
+
+            // Channels/chats are not loaded by default and only load when first put active
+            active: model.itemId && root.isSectionActive &&
+                    (model.itemId === root.activeChatId || model.itemId === root.activeSubItemId)
             width: parent.width
             height: parent.height
+            visible: model.active
+
+            // Removing the binding in order not to unload the content:
+            // It is done for keeping:
+            //   - the last channel/chat scroll position
+            //   - the last typed but not sent text
+            onStatusChanged: if (status == Loader.Ready) active = true
```

Comment:
Just came up with this trick, it should result in the same behavior as `onStatusChanged` binding breakage. 
```qml
Binding on active {
    when: !chatLoader.active
    value: model.itemId && root.isSectionActive && (model.itemId === root.activeChatId || model.itemId === root.activeSubItemId)
}
```

Reply:
changed